cves/CVE-2017-12615.yml

```diff
@@ -76,7 +79,7 @@ incomplete_fix_instructions: |
   Please list any fix commits for this vulnerability that had to be corrected
   at a later date.
 incomplete_fixes:
-  - commit:
+  - commit: e862b0c259b4c867553df29cef1af9e904af83b0
     note:
```

Comment:
why was this fix not complete? Was there any commit message with this specific commit?

Reply:
From here: https://svn.apache.org/viewvc?view=revision&revision=1804604
The commit says that this is only part of the fix for this CVE. The fix was completed later in another revision (listed in fixes).